src/main/java/org/gridsuite/ds/server/service/DynamicSimulationWorkerService.java

```diff
@@ -138,6 +143,16 @@ public Consumer<Message<byte[]>> consumeRun() {
                 if (e instanceof InterruptedException) {
                     Thread.currentThread().interrupt();
                 }
+            } finally {
+                // clean temporary directory created in the config dir by the ParametersService
```

Comment:
@thangqp Not easy to do better now, but this is quite weak design as the working directory has been created by ParameterService called by DynamicSimulationService (so before going through the queue) and cleanup is done in a totally different part (worker part) which could be hard to understand. But this is a temporary design so ok...

Reply:
Yes, every things related to create/remove tempo directory must be removed in the next phase of dynawaltz API